### PR TITLE
Docs/new external lib document

### PR DESCRIPTION
**Pull request by ganesh-bruno**
This PR will add following changes:
- Added `spectral` api 
- new document for extrnal lib

### CODE REVIEW

File: src/pages/testing/script/external-libraries.mdx [hunk 1]
```diff
@@ -1,35 +1,80 @@
...
+
+## Prerequisites
+Before you begin, ensure that you have the following:
+
+1. Node.js installed on your system.
+2. Bruno installed.
+
+## Project Structure
+
+The folder structure for this example will look as follows:
```
Comment (rreyn-bruno) [inline src/pages/testing/script/external-libraries.mdx]:
```suggestion
The folder structure for this example will be:
```
```suggestion
The folder structure for this example will look as follows:
```

File: src/pages/testing/script/external-libraries.mdx [hunk 2]
```diff
@@ -1,35 +1,80 @@
...
 ```
 
-And then you can require the node module in your script as usual.
-**Example:**
-```js
-const { faker } = require('@faker-js/faker');
 
-const randomName = faker.name.fullName();
-const randomEmail = faker.internet.email();
+Run this command into your terminal to initialize `.spectral.yml` file which is predefined ruleset.
```
Comment (rreyn-bruno) [inline src/pages/testing/script/external-libraries.mdx]:
```suggestion
Run this command in your terminal to initialize `.spectral.yml` file which is predefined ruleset.
```

File: src/pages/testing/script/external-libraries.mdx [hunk 3]
```diff
@@ -1,35 +1,83 @@
...
+    <FileTree.File name="bruno.json" />
+    <FileTree.Folder name="node_modules">
+      <FileTree.File name="spectral-cli" />
+    </FileTree.Folder>
+  </FileTree.Folder>
+</FileTree>
+
+## Configuration You need to initialize `package.json` file inside your collection.
+
+Navigate to your Bruno collection directory.
```
Comment (anusree-bruno) [inline src/pages/testing/script/external-libraries.mdx]: Shouldnt we mention in terminal navigate to the collection directory?

File: src/pages/testing/script/external-libraries.mdx [hunk 4]
```diff
@@ -1,35 +1,83 @@
...
+```bash
+npm init -y
 ```
 
-And then run npm install inside your collection folder
+Now you can install any npm package or module to use in your collection.
+
+## Spectral Integration
+
+In this example we're using [Spectral CLI](https://github.com/stoplightio/spectral?tab=readme-ov-file) as a npm package which help you to valide OpenSpec API YAML or JSON files with the built-in ruleset.
```
Comment (anusree-bruno) [inline src/pages/testing/script/external-libraries.mdx]: valide? Please do a spell check.

**Review by lohit-bruno — DISMISSED** [review]

**Review by lohit-bruno — APPROVED** [review]

**Pull request merged**